pkg/secretless/plugin/sharedobj/available_plugins.go

```diff
@@ -59,35 +59,30 @@ func AllAvailablePluginsWithOptions(
 		return nil, err
 	}
 
+	// Assemble HTTP plugins
 	httpPlugins := map[string]http.Plugin{}
-
 	for name, httpPlugin := range internalPlugins.HTTPPlugins() {
-		if _, ok := httpPlugins[name]; ok {
-			// TODO: Should this ever happen?  Do we need this check?  Should it panic?
-			logger.Warnf("Internal plugin '%s' replaced by internal plugin", name)
-		}
+		// Since we're ranging over a map with plugin name as key,
```

Comment:
the key point is that the map is hardcoded, and we create it - the only way for duplicate names to happen is if we erroneously add the same name twice

your comment sort of gets to that, but it could be a bit more clear about how there is no possibility

Reply:
I've reworded the comment... hopefully it's clear enough, but please let me know if it can be clearer.